reports/default/json-schema.json

```diff
@@ -6057,6 +6057,727 @@
+                                                "fillable_by": [
+                                                    "service.history"
+                                                ]
+                                            },
+                                            "end": {
+                                                "description": "Дата окончания действия страхования авто",
+                                                "oneOf": [
+                                                    {
+                                                        "type": "null"
+                                                    },
+                                                    {
+                                                        "$ref": "#/definitions/datetime"
+                                                    }
+                                                ],
+                                                "examples": [
+                                                    "2019-01-22 00:00:00"
+                                                ],
+                                                "fillable_by": [
+                                                    "service.history"
+                                                ]
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                            "client": {
+                                "type": "object",
+                                "additionalProperties": false,
+                                "properties": {
+                                    "type": {
+                                        "description": "Тип клиента",
+                                        "type": [
+                                            "string",
+                                            "null"
+                                        ],
+                                        "examples": [
+                                            "PERSON",
+                                            "LEGAL",
+                                            "LEASING",
+                                            null
+                                        ],
+                                        "fillable_by": [
+                                            "service.history"
+                                        ]
+                                    }
+                                }
+                            },
+                            "geo": {
+                                "description": "Гео-данные места нахождения СТО",
+                                "oneOf": [
+                                    {
+                                        "type": "null"
+                                    },
+                                    {
+                                        "$ref": "#/definitions/geo_raw_data"
```

Comment:
`geo` field contain only `region` and `city`, and `geo_raw_data` describes more fields. We can't use this definition for this field.

Reply:
Ok, if we add all fields from `geo` object to report content, this will be ok. It's approved by project manager. Resolved.

Reply:
Maybe make like https://github.com/avtocod/specs/pull/76/files#diff-0a41a92c2b2f3e904e4546934e7068a1R909-R933 ?

Reply:
@tarampampam 
We no need to do this. Problem was that source gives data only for `region` and `city` fields of `geo` object. At first, we wanted to add only these fields to the report. In this case it would be wrong to use definition for `geo` object. And finally we decide to extend `geo` object in the report for compatibility with this definition, but we'll fill only `region` and `city` fields.